dashboard/app/jobs/aichat_request_chat_completion_job.rb

```diff
@@ -47,19 +47,11 @@ def perform(request:, locale:)
     request.update!(response: response, execution_status: status)
   end
 
-  private def get_toxicity_threshold_user_input
-    DCDO.get("aichat_toxicity_threshold_user_input", DEFAULT_TOXICITY_THRESHOLD_USER_INPUT)
-  end
-
-  private def get_toxicity_threshold_model_output
-    DCDO.get("aichat_toxicity_threshold_model_output", DEFAULT_TOXICITY_THRESHOLD_MODEL_OUTPUT)
-  end
-
   private def get_execution_status_and_response(model_customizations, stored_messages, new_message, level_id, locale)
     # Moderate user input for toxicity.
     # get_toxicity returns an object with the following fields:
     # text: string, toxicity: number, and max_category {name: string, score: number}
-    user_toxicity = find_toxicity('user', new_message[:chatMessageText], get_toxicity_threshold_user_input, locale)
+    user_toxicity = AichatSafetyHelper.find_toxicity('user', new_message[:chatMessageText], locale)
```

Comment:
You mentioned that follow-up work will be to remove `aichat`-specific profanity code in `files_api.rb`. If we only want sources to be checked here, we could set the DCDO flag `labs_to_check_for_profanity` as an empty array for now.
https://github.com/code-dot-org/code-dot-org/blob/b458b4980113bb971a63aad6c574574a2cb1615d/dashboard/legacy/middleware/files_api.rb#L24

Reply:
Good call! removed from this list